prov/rxm/src/rxm_ep.c

```diff
@@ -633,6 +635,131 @@ static ssize_t rxm_rma_iov_init(struct rxm_ep *rxm_ep, void *buf,
 	return sizeof(*rma_iov) + sizeof(*rma_iov->iov) * count;
 }
 
+static inline ssize_t
+rxm_ep_transmit_generic_lightweight(struct rxm_ep *rxm_ep, struct rxm_conn *rxm_conn,
```

Comment:
nit: This function is slightly misnamed.  It doesn't actually 'transmit', but formats the packet.

Reply:
I agree, It seems that `rxm_ep_transmit_format_lightweight` and `rxm_ep_transmit_format` should be okay